2_process.smk

```diff
@@ -0,0 +1,155 @@
+import os
+import numpy as np
+import pandas as pd
+
+configfile: "2_process/process_config.yaml"
+
+
+def mntoha_lake_sequence_files(wildcards):
+    """
+    List all MNTOHA lake sequence .npy files for training and testing.
+
+    :param wildcards: Snakemake wildcards (empty).
+    :returns: List of lake training/testing sequence files.
+
+    """
+    # make this function dependent on fetch_all
```

Comment:
`fetch_all` is no longer a rule, right?

Reply:
Right - that comment is out of date. It should read: 
```
# make this function dependent on fetch_mntoha_metadata
```